components/org.wso2.carbon.identity.application.authentication-endpoint/src/main/webapp/Session.jsp

```diff
@@ -0,0 +1,310 @@
+                                                <td><%=remoteIp%>
+                                                </td>
+                                                <td><%= startedTime %>
+                                                </td>
+                                                <td><%=lastAccessedTime%>
+                                                </td>
+                                            </tr>
+                                                <%
+                                                        index++;
+                                                        }
+                                                    %>
+                                        </table>
+                                        </br></br>
+                                        <div align="Center">
+                                            
+                                            <button
+                                                    onclick="$('#loading').show();" class="btn btn-primary">
+                                                Terminate and Proceed
+                                            </button>
+                                        </div>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                    </form>
+                
+                </div>
+            </div>
+            <!-- /content -->
+        
+        </div>
+        
+        <!-- /content/body -->
+    
+    </div>
+</div>
+
+<!-- footer -->
+<footer class="footer">
+    <div class="container-fluid">
+        <p>WSO2 Identity Server | &copy;
+            <script>document.write(new Date().getFullYear());</script>
+            <a href="http://wso2.com/" target="_blank"><i class="icon fw fw-wso2"></i> Inc</a>. All Rights Reserved.
+        </p>
+    </div>
+</footer>
+<script src="libs/jquery_1.11.3/jquery-1.11.3.js"></script>
+<script src="libs/bootstrap_3.3.5/js/bootstrap.min.js"></script>
+</body>
+</html>
+<%!
+    private static final String DEFAULT_USER_AGENT_NAME = "Unknown";
+    private static final SimpleDateFormat DATE_FORMAT = new SimpleDateFormat("HH:mm  EEE, d MMM yyyy ");
+    
+    //Method to identify time difference between current time and session creation time
+    private String getTimeDifference(long currentTime, long startedTime) {
+        
+        long timeDiff = currentTime - startedTime;
+        String lastAccessed = null;
+        if (timeDiff < 60000) {
+            lastAccessed = String.valueOf(round(timeDiff / 1000)) +
+                    " Seconds ago";
+        } else if (timeDiff < 3600000) {
+            lastAccessed = String.valueOf(round(timeDiff / 60000)) +
+                    " Minutes ago";
+        } else if (timeDiff < 86400000) {
+            lastAccessed = String.valueOf(round(timeDiff / 3600000)) +
+                    " Hours ago";
+        }
+        return lastAccessed;
```

Comment:
Better to add a comment mentioning the logic behind this. And we can add the equation, which we got these numbers

Reply:
comments added to describe the logic